defi-pulse/src/symbols/index.ts

```diff
@@ -0,0 +1,57 @@
+import fs from 'fs'
+import { join } from 'path'
+import { ethers, utils } from 'ethers'
+import { util } from '@chainlink/ea-bootstrap'
+import { logger } from '@chainlink/external-adapter'
+
+const rpcUrl = util.getRequiredEnv('RPC_URL')
```

Comment:
We have the same problem here, this config is buried too deep and leaves us little options in the future to change it for something else if the situation requires. This should be a function param passed from the top.